oat-sa/extension-tao-testqti
# fixed message for general error of the messages action

Pull request summary
 
Related to : https://oat-sa.atlassian.net/browse/TCA-523
 
Synchronization service was not sending all requested responses in case of failure (it stopped on the first failure, so all other actions weren't answered)
 
#### How to check
 - create delivery with proctoring (backoffice)
 - run test as a test taker (via lti)
 - made change on the backend to have an error in the sync request:
![2020-05-04_15-40-14](https://user-images.githubusercontent.com/1445911/80969638-9c44ac80-8e22-11ea-8c8c-c9f2337381e1.png)

 - pause the test as a proctor
 - try to press okay button in the popup window, and you will see answer:
![2020-05-04_15-41-56](https://user-images.githubusercontent.com/1445911/80969685-abc3f580-8e22-11ea-86a0-e96187a6b405.png)
